src/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -361,6 +345,20 @@ private async Task<bool> HandleTaskDestinations(WorkflowInstance workflowInstanc
 
             foreach (var taskExec in taskExecutions)
             {
+                if (string.Equals(taskExec.TaskType, TaskTypeConstants.RouterTask, StringComparison.InvariantCultureIgnoreCase))
+                {
+                    processed &= await HandleTaskDestinations(workflowInstance, workflow, taskExec, correlationId);
+
+                    if (!processed)
```

Comment:
small point, but I prefer `if (processed is false)`

Reply:
Changed